src/screens/StartScreen/StartScreen.js

```diff
@@ -5,6 +5,7 @@ import LoginSignupScreen from '../LoginSignupScreen/LoginSignupScreen';
 import { Spinner, Button } from '../../components/common';
 import { firebase } from '../../config/config';
 import HomeScreen from '../HomeScreen/HomeScreen';
```

Comment:
If you see any unused imports, please remove them.

Reply:
Okay!